execution/executor/src/components/apply_chunk_output.rs

```diff
@@ -167,27 +167,15 @@ impl ApplyChunkOutput {
         let StateCache {
             frozen_base,
             mut accounts,
+            mut state_cache,
             proofs,
         } = state_cache;
 
         // Apply write sets to account states in the AccountCache, resulting in new account states.
-        let account_states = to_keep
+        let state_store_updates = to_keep
             .iter()
-            .map(|(t, o)| process_write_set(t, &mut accounts, o.write_set().clone()))
-            .collect::<Result<Vec<_>>>()?;
-        let state_store_updates = account_states
-            .par_iter()
```

Comment:
Nice that we'll be able to remove this costly serialization when we go fine grained.